services/markup/processorhelper.go

```diff
@@ -14,6 +14,9 @@ import (
 func ProcessorHelper() *markup.ProcessorHelper {
 	return &markup.ProcessorHelper{
 		IsUsernameMentionable: func(ctx context.Context, username string) bool {
+			if ctx == nil {
+				return false
```

Comment:
According to Go's behavior: nil ctx should  always cause panic.

Hiding bugs would cause more  bugs.

Reply:
Sorry for this mistake. I added this check for test and forgot to remove it. I have removed these code.